### ISSUE TITLE
[feature suggestion] Repetitive notifications when overdue

### ISSUE THREAD

**Issue by thisago** [Medvedev91/timetome#56]
Hello Ivan, first thanks for the awesome app, I loved the idea of the non stoppable timer.

I would like to suggest adding an optional repetitive notification alerts to remind that timer is expired, I usually forget it running after overdue, specially when I get too much focused (and sometimes the focus is on procrastination tasks).

Like an reminder to keep eye on time, and to avoid spend too much time on a task without break

Again, thanks for your awesome work!

**Comment by Medvedev91** [comment]
Hi,

Thank you for the feedback!

Earlier versions had multiple notifications. In real life, it just learns to ignore notifications.

I plan to implement persistent notifications. I believe it will work better. Related issue #15.

**Comment by thisago** [comment]
Yeah, that's right, it's really an problem get the habit to ignore notifications, persistent notification could help indeed!

Sorry about possible duplicate. And thanks for the quick reply!

**Comment by Medvedev91** [comment]
> Sorry about possible duplicate.

Don't worry, community feedback is what motivates. Feel free to create issues.
